ERC721/serverless-NFT/part-2.md

```diff
@@ -0,0 +1,536 @@
+
+Add it to the app officially and we can see it in the rinkeby section.
+
+![](https://ucb4f2d0fe014a038f5db3d4351c.previews.dropboxusercontent.com/p/thumb/AAQyRq7FC8lQ59Z7tPl3K35Z-qjtFHr6AHqs9wPkRcDIrcp9fI3yIAxLIIAozqtpcA2FLe2ILncNbh6eU4UQRF__miLE-OG3BnxZg0H90hi4QVYqhhLKnmDIGSJF2G2uRxYs5ph55VE7lQvx49ClmAMgw1A-lYKbP_ETmgL9Gidz7PsqCFO5_k6QM5X1zHGrU6C1jQOgNJeEIaXUgUBy889XFbJsBkM9SaoUgQROoEA0lRJjGSCT9Saocq4IEEokTUv7NSaRZmT-6b_sl-RfYFJm0GcTkjN4OYXNENYuWJwf1GX2b5AA40RJ8RZwmdiDrG7kP7aiAyyRgUrHcvPXBX3J/p.png?size=1600x1200&size_mode=3)
+
+![](https://uc6721b9fbe8b9e400b0e16a2b2d.previews.dropboxusercontent.com/p/thumb/AAQeFmkQqEaiy9bAlkIy2FV4UsG0vBotkFZ0_Dzn0mh6O2DCuJinbK-n7zlZ1_gdzmI8PBPsSVezc5Iu2vWsSRY1buw7k-a4CNJGUodJ8TvPsnLf55TFwqcymHtVY5v4UmnSBXbTRb3PAnb0TqmlboscKyMFREn-P3eMeKGMKe8y9Gm0Cm8uy-YpWLNLXh1XCzuq8t8UL17bKn3GY7XQILxb_UGSH6CLm90YeIwKRPAVEkjOjbAa0zOwscawB0MnluWXY1uB4snGafdzTUAY9_VGFbh04KM-txa04VC8HAfx4pjF4IVG7pX5h3yP-Itz3KBDK47l0itBTasaLj4oNhfh/p.png?size=1600x1200&size_mode=3)
+
+![](https://ucd9b2861e6e5ca875b4b34c1bc3.previews.dropboxusercontent.com/p/thumb/AAS4FZYKNCr6iW3NaN7sjFFwcZXdod24D8n3DRHq91n0AT8T_aL63FEtjpZXQIHC1o62wnN6ecLfatgZ_1MfopIsfPsMDQUxwE2ppprAnK31XzbeXxocj_4KzxAMSxK1Plg_aeNRYv_gzB7xG4zki_86iPks53c5kOPv8r2IAepGjbmXQUxfW6J6CO_szBAw3K5OiY8FEZ7B6a5UxWzLtbDiIOFrvVrQuUyH5BxWb7caZ08acxtA74Z2_0Hv-Bb68CTVi9uLd5Hma4sga1bXtebf32KDFG_4jPX91zsJSsIyRcneOYrjA7kWHgnvAmhmzdBYmCiNy5gJKZusVNz-OIzq/p.png?size=1600x1200&size_mode=3)
+
+And we're up!
+
+![](https://uc4202e9fca8d3904326cce90f88.previews.dropboxusercontent.com/p/thumb/AASwwB6oLcA9dncl3v72gavD8lsLJgfjAKLIffLx__UvTaC5ojT_et1GnBS1wo5jBCarhHlgrKUCh2MPA6M2wgZw50L-nkco8VAqPJm7X1E3jEiWR6un59p7eW_vzEhWwzKqza13BbHckTOcBbopdM4vnHrz4cmYQ0M4UXUMRXrLIj_C3TkSDOMRt5osnw57zsjwftH47VveoDodLY9ghKQMKEbeiJoPsmXOugTuPau8jVMpk78PJFfM0AJXDlw5_xsQ-rhGyKE5XSQF8VF9YqSZky4wNDumPeM7wPqmLR9FWpZk6yjhe8Y1hiBJcpeTwB1oidz2KQmfhABiWRncfhOv/p.png?size=1600x1200&size_mode=3)
+
+Let's add it to rarebits too.
+
+![](https://ucab4fda645c20b2e7fa248ba66c.previews.dropboxusercontent.com/p/thumb/AAR1iLY0W1VWdK4sYz2n-c7MbZq_jAIHXOIPZeq6vAhUV4NX_uJQ3mSLJISnnE9DuDbJKPwb3l4Kc0fTCUFx0hKPUguXg-iAArEB31U7uP2YawpODPAYuVUkKw_1jfHsz0BkrcVc_UkhTin_cqhHpF5aYvHOMFh9LrO-KihL0T4CjOcvKWNDfOrzrlbzy8pfp3NHzLiN9hCfYPnv5MirGv7NwdazkjueDPKxh_17axql-rsMDUpoMZiZLCo_bnIJ_ksH73WUtGHTbrka565XA66Se6ImDK4nJQRGZb7N-DC4_3Wyd-NjxdmSjkzHvg9rFOnDDG0mzOHsZK1fOVT5REb-/p.png?size=1600x1200&size_mode=3)
+
+![](https://ucb4894f15b87e1d4d2a0f6f9cb9.previews.dropboxusercontent.com/p/thumb/AATeH3f78SkisPzM0BGUke0SSL2PcPKeUYvjB9YPr8IwEmZRiZ8XKWAnCdpmD6_DFb5Mg5ray985x0yaG-5TkxrgRACnqdbN5JYi8Gwom05hnUoIBTnWCYHTSlZfV6k9gde_515kIGIYFcgFF7s-0zwm1IDZ6UEx8jhmMpl7Y4-2TiMHPF_fORqxTzBi2AOYS_UxlBoOjPSlV1YOws8Vpj51M5SFpkq1-GdSpn9rh9CTe71iKSRSBFrR4QwamHPI93s7cGPfZE17OyeFWiMiPvskG4L5PxPIDl278PMH5jJv5FdWaHDqfIGGQgPI3q2oCQpsfLawMH0XJnskp3yOFYwX/p.png?size=1600x1200&size_mode=3)
+
+## Next Steps
```

Comment:
@okwme Mostly great and done. I wondered if you had some other recommended next steps we could add here for improvements and for further resources?